README.rdoc

```diff
@@ -4,10 +4,14 @@
 
 == DESCRIPTION:
 
-A gemified version of Syck from Ruby's stdlib.  Syck has been removed from
-Ruby's stdlib, and this gem is meant to bridge the gap for people that haven't
+This Gem is a wrapper around {Syck}[https://github.com/indeyets/syck] (a.k.a. Scripters' YAML Cobble-Yourself-a-Parser Kit),
```

Comment:
Dunno if there's any standard for markdown content in projects under the `ruby` org but perhaps this text should try to match the 80-column wrapping of the original content?

Reply:
Done! Agree it's probably not necessary in this day and age, but changed nonetheless